contracts/base/DAIPool.sol

```diff
@@ -89,20 +109,130 @@ contract DAIPool is DAIPoolInterface, Initializable {
         emit DaiDeposited(msg.sender, amount);
     }
 
-    function withdrawDai(uint256 amount) external {}
+    /**
+        @notice It allows any zDAI holder to burn their zDAI tokens and withdraw their DAIs.
+        @param amount of DAI tokens to withdraw.
+        @dev It throws a require error if the contract hasn't enough DAI balance.
+        @dev It throws a require error if the holder hasn't enough zDAI balance.
+     */
+    function withdrawDai(uint256 amount) external {
+        // Checks contract hast enough DAI balance.
+        requireEnoughDAIBalance(amount);
```

Comment:
DAI balance check is not needed, as all ERC20 tokens revert on `transfer` if not enough balance is provided.